app/models/caution_flag.rb

```diff
@@ -2,6 +2,9 @@
 
 class CautionFlag < ApplicationRecord
   belongs_to :institution, counter_cache: :count_of_caution_flags
+  scope :unique_flag, lambda {
+    select('DISTINCT title, description, link_text, link_url')
```

Comment:
I think this would be more rails-ish as `select('title, description, link_text, link_url').distinct`

Reply:
changed